src/worker.js

```diff
@@ -0,0 +1,14 @@
+import { runAsWorker } from 'synckit';
+
+runAsWorker(async ({
+  text, prettierEslintPath, filePath, extensionConfig,
+}) => {
+  // eslint-disable-next-line import/no-dynamic-require, global-require
+  const format = require(prettierEslintPath);
+  return format({
+    text,
+    filePath,
+    logLevel: 'info',
+    prettierLast: extensionConfig?.prettierLast || false,
```

Comment:
```suggestion
    prettierLast: extensionConfig?.prettierLast,
```

Personally, I think we should not add such useless default value.